pkg/verify/certificate_identity.go

```diff
@@ -91,6 +91,10 @@ func (s SubjectAlternativeNameMatcher) Verify(actualCert certificate.Summary) bo
 }
 
 func NewCertificateIdentity(sanMatcher SubjectAlternativeNameMatcher, extensions certificate.Extensions) (CertificateIdentity, error) {
+	if sanMatcher.SubjectAlternativeName.Type == "" && sanMatcher.SubjectAlternativeName.Value == "" && sanMatcher.Regexp.String() == "" {
```

Comment:
Should we exclude `Type` from this check, and just require `Value` or `Regexp`?

`Type` seems more like a detail to cover specific cases where various SANs might look similar, rather than an identity string.